Makefile

```diff
@@ -20,11 +20,11 @@
 #==================================================================================================
 
 
-CPP                =
+CPP                = 
 PYTHON             = python
 COMPILER_MODE      = DEBUG
 PRECISION          = DOUBLE
-OPENMP             = 1
+OPENMP             = 0
```

Comment:
Also, if I don't change the compiler (e.g. leaving it blank) and I just switch on MPI, will it work? On my machine compilation succeeds (the compiler can find the MPI headers) but linking does not (the linker cannot find the MPI library). I guess what is going to happen depends on the specific setup, but it might be confusing that one switches on MPI and actually it doesn't work. What do you think? I propose the following:
a) if CPP is set to mpic++ (or variations, i.e. mpicpp, mpicxx, mpiCC), MPI is set to 1 automatically. This is needed for people that do have the compiler wrappers so that they have to change one variable in the Makefile rather than two
b) we explain clearly in the userguide that if you switch on MPI and compilation fails, you need to change CPP as well. There is no way of doing this automatically as we can't know the right compiler to use on the user's machine...